report/4.Design/Conclusion.tex

```diff
@@ -15,12 +15,11 @@ \section{Conclusion}
 These decisions will help shape the physical aspects of the solution.
 
 
-\paragraph{Algorithms}
-%TODO write the specific algorithm that we decide upon using. also make sure  this is actually up to date [ask thomas]
-GOTURN was chosen as the default machine-learned algorithm.
-However, using a learned algorithm, can be rather ineffective in these cases, so each specific algorithm will have different pros and cons.
-Therefore, the reasoned solution was to implement multiple algorithms in a modular fashion, so they can be used interchanged.
+\paragraph{Object localization}
+Unfortunately due to hardware limitations, no machine learning algorithm will be fast enough to be feasible in the project.
+Therefore the fall back solution is to use the "Thresh Moment" algorithm, since it performs exceptionally well in terms of both performance and accuracy, while being very simple due to the utilization of the OpenCV functions.
```

Comment:
Maybe something has to be added, but the purpose is to state that the algorithm itself is very simple, since it utilizes a lot of abstraction. This obviously also implies it is easy to implement, but first of all the algorithm in itself is simple. Any thoughts?